edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/NestedDeviceScopeApiClient.cs

```diff
@@ -165,10 +165,6 @@ async Task<ScopeResult> GetIdentityOnBehalfOfInternalAsync(Uri uri, string devic
                 Option<string> maybeAuthChain = await this.serviceIdentityHierarchy.GetAuthChain(onBehalfOfDevice);
                 string authChain = maybeAuthChain.Expect(() => new InvalidOperationException($"No valid authentication chain for {onBehalfOfDevice}"));
 
-                // We must pass the origin Edge ID along upstream, so the root
-                // can know which Edge to act OnBehalfOf when calling IoT Hub
-                msg.Headers.Add(Constants.OriginEdgeHeaderKey, onBehalfOfDevice);
```

Comment:
>msg.Headers.Add(Constants.OriginEdgeHeaderKey, onBehalfOfDevice); [](start = 16, length = 65)

please have someone make sure that adding a newly created leaf at the bottom level still works.  This is covered by the nested E2E tests, but wouldn't hurt to have someone run it manually at least once.